samples/quick-start/docker-compose.yml

```diff
@@ -1,9 +1,11 @@
-version: '3'
+version: '3.5'
 services:
   eventstore:
-    image: eventstore/eventstore:20.10.0-buster-slim
+    image: eventstore/eventstore:latest
     environment:
-      - EVENTSTORE_INSECURE=true
-      - EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP=true
```

Comment:
this line is needed for the stream browser in the UI to work. could be useful in the quickstart

Reply:
fixed